edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html

```diff
@@ -0,0 +1,163 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Awaiting Proctoring Software Installation and Setup
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      You have chosen to take {{display_name}} as a proctored exam. You should be directed to a new window
```

Comment:
@chrisndodge 
1) Add quotation marks around the exam display name.
2) Delete the "You should be directed to a new window..." sentence.
3) Change the 3rd sentence to: "Open the proctoring software installation and setup window."